NAudio.Lame/LameConfig.cs

```diff
@@ -128,6 +146,17 @@ public LameDLLWrap.LibMp3Lame ConfigureDLL(WaveFormat format)
 			if (UseFreeFormat != null) result.UseFreeFormat = UseFreeFormat.Value;
 			if (VBR != null) result.VBR = (LameDLLWrap.VBRMode)VBR.Value;
 
+			// VBR Control
+			if (VBRMinimumBitRate != null) result.VBRMinBitrateKbps = VBRMinimumBitRate.Value;
+			if (VBRMaximumBitRate != null) result.VBRMaxBitrateKbps = VBRMaximumBitRate.Value;
+			if (VBRForceMinimum != null) result.VBRHardMin = VBRForceMinimum.Value;
+			if (VBRAlgorithmQuality != null)
+			{
+				// ignore settings that are out of range
+				if (VBRAlgorithmQuality.Value >= 0 && VBRAlgorithmQuality.Value <= 9)
+					result.VBRQualityLevel = VBRAlgorithmQuality.Value;
+			}
```

Comment:
This can be simplified.  Value comparisons to `null` are fine:

    if (VBRAlgorithmQuality >= 0 && VBRAlgorithmQuality <= 9)
        result.VBRQualityLevel = VBRAlgorithmQualityLevel.Value;